ui-tests/src/test/java/org/openmrs/reference/page/EditVisitNotePage.java

```diff
@@ -10,12 +10,24 @@
 package org.openmrs.reference.page;
 
 import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
 
 public class EditVisitNotePage extends VisitNotePage {
-
+    private static final By NOTE = By.id("w10");
+    String note = "this is edited note";
```

Comment:
Since `note` is only used within `EditVisitNotePage` class then making it private static final variable is  a better deal.